pkg/controller/servicebinding/sbrcontroller.go

```diff
@@ -43,8 +44,8 @@ var _ ResourceWatcher = (*sbrController)(nil)
 // controllerName common name of this controller
 const controllerName = "servicebinding-controller"
 
-// compareObjectFields compares a nested field of two given objects.
-func compareObjectFields(objOld, objNew runtime.Object, fields ...string) (bool, error) {
+// compareRuntimeObjectFields compares a nested field of two given runtime objects.
+func compareRuntimeObjectFields(objOld, objNew runtime.Object, fields ...string) (bool, error) {
```

Comment:
why do we need to rename this function?

Reply:
I had planned to use this function for ObjectMeta as well, but then later realized that only Runtime Object is the param for this function. So to avoid confusion in the future I added the Runtime as a prefix to the Object.